torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13285,7 +13285,7 @@ def ref_pairwise_distance(input1, input2):
     OpInfo('sort',
            dtypes=all_types_and(torch.bool, torch.float16, torch.bfloat16),
            dtypesIfCUDA=all_types_and(torch.float16, torch.bfloat16),
-           dtypesIfROCM=all_types_and(torch.float16),
+           dtypesIfROCM=all_types_and(torch.float16, torch.bfloat16),
```

Comment:
Like the argsort, we can probably remove `dtypesIfROCM` from here and in `msort`

Reply:
agree, better to keep in uniform